packages/node-cli/src/core/sdksetup/SdkProperties.js

```diff
@@ -28,7 +28,9 @@ class SdkProperties {
 	}
 
 	getSdkFileName() {
-		return this.configFileExists() ? CONFIG_FILE_CACHE.sdkFilename : SDK_FILENAME;
+		// read config.js file if exists or use package.json
+		const configFile = this.configFileExists() ? CONFIG_FILE_CACHE : require(PACKAGE_FILE);
+		return configFile.sdkFileName;
```

Comment:
Should we do something like this instead?
`const sdkFileName = this.configFileExists() ? CONFIG_FILE_CACHE.sdkFileName : sdkFileName`

Reply:
This will require a refactor of TC build configs also right? because we are generating config files on the fly, with `sdkFilename`, so if we change that to `sdkFileName` it will cause issues I guess.

Previous I did something like that, 

`this.configFileExists() ? CONFIG_FILE_CACHE.sdkFilename : sdkFileName`  but it was looking weird

Either we make both `sdkFileName` or `sdkFilename`
and refactor TC configs also accordingly. What do you think? @karl-anthony-ng

Reply:
No, what I meant here is that you already imported sdkFilename from package.json so there's no need to require again. You can just reference it directly. Just a mistake on the casing I made in the comment.
`const sdkFilename = this.configFileExists() ? CONFIG_FILE_CACHE.sdkFilename : sdkFileName`

And maybe we should also use sdkFilename in package.json instead of sdkFileName to make it consistent?